### PR TITLE
Updated .gitignore to allow CMake on VSCode to generate build files

### PR DESCRIPTION
**Pull request by AJThePro99**

### CODE REVIEW

**Comment by AJThePro99** [comment]
Bro, update pls?

**Review by lokesh-tr — COMMENTED** [review]
Can you sort the .gitignore in alphabetical order?

**Comment by AJThePro99** [comment]
Sure! Be on that right away.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+./apps/cmake-build-debug
+/build
```
Comment (lokesh-tr) [inline .gitignore]: It would be nice if you follow the convention...
The directories should be prefixed with ./

**Review by lokesh-tr — CHANGES_REQUESTED** [review]
Oops, i think you included the build directory some how.

**Comment by lokesh-tr** [comment]
I guess the commit history became too clumsy... Can you create clean your commits / create a new branch and open a new PR?

**Pull request closed without merging**